### DISCUSSION TITLE
Unable to change music cover images

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5971] [Q&A]
Hi!
I noticed that on the latest versions, it is impossible to modify the cover of a music, the latter is simply not applied. I tried to create a new fresh installation of Azuracast, the same problem occurred...

Currently, I'm on the Rolling Release build, anyone noticed the same issue?

**Comment by ivellios1988** [comment]
Does it show any error messages when trying to update the cover arts? I see 'out of storage' when I try to update mine. Even though I have whole gigabytes of free disk space.

**Reply by Mozitux** [reply]
Nothing, but since the last update, my issue seems to be fixed
